core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -114,6 +134,33 @@ public HubResponse<Enrollee> enroll(ParticipantUser user, PortalParticipantUser
         return hubResponse;
     }
 
+    private Enrollee enrollGovernedUser(String portalShortcode, ParticipantUser proxyUser, StudyEnvironment studyEnv, UUID preEnrollResponseId) {
+        // Before this, at time of registration we have registered the proxy as a participant user, but now we need to both register and enroll the child they are enrolling
+        RegistrationService.RegistrationResult registrationResult = registrationService.registerGovernedUser(portalShortcode, proxyUser);
+        Portal portal = portalService.findOneByShortcode(portalShortcode).orElseThrow();
+        log.info("Governed user {} registered", registrationResult.participantUser().getId());
+        ParticipantUser governedUser = registrationResult.participantUser();
+        PortalParticipantUser ppMainUser = registrationResult.portalParticipantUser();
+        Enrollee enrollee = Enrollee.builder()
+                .studyEnvironmentId(studyEnv.getId())
+                .participantUserId(governedUser.getId())
+                .profileId(ppMainUser.getProfileId())
+                .preEnrollmentResponseId(preEnrollResponseId)
+                .build();
+        enrollee = enrolleeService.create(enrollee);
+
+        EnrolleeRelation enrolleeRelation = EnrolleeRelation.builder()
+                .enrolleeId(enrollee.getId())
+                .participantUserId(proxyUser.getId())
+                .relationshipType(RelationshipType.PROXY)
+                .portalId(portal.getId())
+                .build();
+
+        enrolleeRelationService.create(enrolleeRelation);
+        return  enrollee;
```

Comment:
for clarity and easier testing, I would create a record that contains everything this service method created, and then return that.
```suggestion
        return  new GovernedEnrolleeCreationRecord(enrollee, enrolleeRelation, governedPpUser, governedUser);
```